rustls/src/msgs/message_test.rs

```diff
@@ -72,7 +72,7 @@ fn can_read_safari_client_hello() {
     let mut rd = Reader::init(bytes);
     let m = OutboundOpaqueMessage::read(&mut rd).unwrap();
     println!("m = {:?}", m);
-    assert!(Message::try_from(m.into_plain_message()).is_err());
+    Message::try_from(m.into_plain_message()).unwrap();
```

Comment:
This is unchanged in your branch but I think this test case might benefit from having the fn renamed from `can_read_safari_client_hello` to something like `can_read_safari_client_hello_bad_sni`, or maybe there could be a comment on `bytes` that indicates the payload includes an invalid SNI of 127.0.0.1?

It's hard to divine that from the blob of hex otherwise :-)